### DISCUSSION TITLE
创建百度底图

### DISCUSSION THREAD

**Discussion by KipSong** [antvis/L7#1908] [Q&A]
通过实现 BaseMapService 类，完成百度地图的方法，但是绘制的 Layer 可能根据拖动偏移，这块高手指点一下；

**Comment by lzxue** [comment]
有没有已完成的实例demo,可以辅助你实现

**Reply by KipSong** [reply]
https://codesandbox.io/s/nameless-smoke-n7drvd?file=/src/App.tsx
谢谢大佬，给个联系方式，让我给你点杯咖啡

**Reply by lzxue** [reply]
我先看一下，一起建设开源生态

**Reply by lzxue** [reply]
线上跑起来有点卡顿，下面本地启动有问题，建议是否可以用 dumi 新疆 L7-BMap 的项目 https://d.umijs.org/guide/initialize 
可以加一下 qqlzxue 微信联系我
